### DISCUSSION TITLE
Add mention from v8.25.0 about deprecated .eslintignore with flat config

### DISCUSSION THREAD

**Discussion by alexey-koran** [eslint/eslint#17342] [Ideas] [answered]
Hello,
Maybe it will be useful to add info that [new config system no longer supports .eslintignore file](https://eslint.org/blog/2022/10/eslint-v8.25.0-released/#highlights) to the [flat config ignore docs section](https://eslint.org/docs/latest/use/configure/configuration-files-new#specifying-files-and-ignores)?
Or to migration guide

**Comment by fasttime** [comment, marked as answer]
Thanks for pointing out this @alexey-koran. There is currently an open [pull request](https://github.com/eslint/eslint/pull/17230) for introducing a migration guide to the flat config. The migration guide contains a section about [ignoring files](https://github.com/eslint/eslint/blob/0059b3c257ff74fff567a984c47e93b115c1edd5/docs/src/use/configure/migration-guide.md#ignoring-files): you could have a look at it. If the provided information doesn't satisfy you, feel free to suggest a change in the pull request.

**Reply by bmish** [reply]
Also, in addition to mentioning this in the appropriate docs, we're going to add a warning when `.eslintignore` is present with flat config:
* https://github.com/eslint/eslint/issues/17831
